factory

```diff
@@ -19,6 +19,9 @@ reference:
 * http://ubuntu-smoser.blogspot.ro/2013/02/using-ubuntu-cloud-images-without-cloud.html
 """
 
+DB_ROOT = Path.home() / '.factory'
```

Comment:
So we're caching images in the user's home folder. It's a good default, but let's at least make it configurable.

Additionally, the readme should say that there's a cache in `~/.factory`.